src/v2/Components/NavBar/NavItemPanel.tsx

```diff
@@ -5,12 +5,12 @@ import styled, { css } from "styled-components"
 export type MenuAnchor = "left" | "right" | "center" | "full"
 
 const AnimatedPanel = styled(animated.div)<{
-  menuAnchor: MenuAnchor
-  offsetLeft?: number
+  "data-menuanchor": MenuAnchor
+  "data-offsetleft"?: number
```

Comment:
This is pretty weird

Reply:
By "pretty weird" I mean — this is just a hack around a symptom of us not updating to the latest version of styled-components for `shouldForwardProp` support.

Reply:
Yeah this was the only way to fix it without having to delete the props from the object (thereby triggering other React warnings). Since data props can be passed in any form it was the most straightfforward path forward (ignoring something like `shouldForwardProp` which i wasn't aware of! we should get that in.)